.github/workflows/deploy.yml

```diff
@@ -34,37 +34,31 @@ jobs:
       - name: Run unit tests
         run: npm run test:ci
 
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
+      - uses: actions/upload-artifact@v2
+        if: github.ref == 'refs/heads/master'
```

Comment:
We're okay with adding artifacts for non-master builds, its the only way to investigate whats going on since unlike some build tools we can't connect to the machine itself to view artifacts.

Reply:
cool all master artifacts will be available, others on push will exist in branch to investigate

Reply:
We want to have artifacts for all workflow runs available though, both master and feature branches.

Reply:
cool stuff, so for every push we back-up artifacts. seems excessive but done..